test_settings.py

```diff
@@ -11,4 +11,9 @@
 ROOT_URLCONF = 'test_urls'
 
 # LMS Urls - for LTI 1.3 testing
-LMS_ROOT_URL = "https://example.com"
\ No newline at end of file
+LMS_ROOT_URL = "https://example.com"
+
+# Dummy FEATURES dict
+FEATURES = {
+    'LTI_1P3_ENABLED': False,
```

Comment:
Just for my own curiosity: why did you need to add this, and why is it False?

Reply:
Instead of relying on the values that the code defaults to, I've decided to explicitly add it here.